### DISCUSSION TITLE
A way to remove object "schema" from webhook payload

### DISCUSSION THREAD

**Discussion by 7thkey** [directus/directus#5973] [Archive]
This was asked before but not answered at all.
https://github.com/directus/directus/discussions/2971

We really need a way to remove the schema object, it is getting so big that our CRM begins to slow down on every contact request.
Is there a way to remove the "schema" object from the payload?

Cheers

**Comment by rijkvanzanten** [comment]
You can use a regular hook to control what exact data is sent out.
Feel free to create a PR to strip the non-payload data from webhooks, as that sounds like it's what you'd expect webhooks to send 👍🏻

**Reply by 7thkey** [reply]
Thanks for the reply @rijkvanzanten
